sites/public/src/pages/index.tsx

```diff
@@ -97,7 +101,7 @@ export default function Home({ latestListings, underConstructionListings }) {
   const metaImage = "" // TODO: replace with hero image
   const alertClasses = "flex-grow mt-6 max-w-6xl w-full"
   return (
-    <Layout>
+    <Layout className={"home-page"}>
```

Comment:
I'm not seeing this in "home-page" defined anywhere in Detroit or Ui-Components

Reply:
Thank you! I had renamed it to `home-page-hero`.